### PR TITLE
Don't throw on 404

### PR DESCRIPTION
**Pull request by matthew-andrews**
@aintgoin2goa

### CODE REVIEW

File: lib/get.js [hunk 1]
```diff
@@ -33,7 +33,11 @@ module.exports = opts => {
 				return decode(data.Item);
 			}
 			metrics.count(`${table}_status_4xx`);
-			throw new Error('URL not found');
+			return {
+				fromURL,
```
Comment (aintgoin2goa) [inline lib/get.js]: is this some kind of ecma6 magic I didn't know about?  why not `fromUrl: fromUrl`?

Comment (matthew-andrews) [inline lib/get.js]: yep

Comment (matthew-andrews) [inline lib/get.js]: `{ fromURL }` <=> `{ fromURL: fromURL }`

Comment (aintgoin2goa) [inline lib/get.js]: nice!

:+1:

Comment (matthew-andrews) [inline lib/get.js]: +1 for the whole PR? :grin:

**Pull request merged**